docs/docs/recipes/styling-css.md

```diff
@@ -395,35 +395,37 @@ Hosting your own [Google Fonts](https://fonts.google.com/) locally within a proj
 
 - A [Gatsby site](/docs/quick-start)
 - The [Gatsby CLI](/docs/gatsby-cli/) installed
-- Choosing a font package from [the typefaces project](https://github.com/KyleAMathews/typefaces)
+- Choosing a font package from [Fontsource](https://github.com/fontsource/fontsource)
 
 ### Directions
 
-1. Run `npm install typeface-your-chosen-font`, replacing `your-chosen-font` with the name of the font you want to install from [the typefaces project](https://github.com/KyleAMathews/typefaces).
+This is an installation example using Open Sans, applicable to all other fonts searchable via [NPM](https://www.npmjs.com/) or the [packages](https://github.com/fontsource/fontsource/tree/master/packages) directory in the repository.
 
-An example to load the popular 'Source Sans Pro' font would be: `npm install typeface-source-sans-pro`.
+1. Run `npm install fontsource-open-sans` to download the necessary package files. If you wish to use a different typeface, search up `fontsource-<insert font name>` on [NPM](https://www.npmjs.com/) for specific instructions tailored to that font.
 
-2. Add `import "typeface-your-chosen-font"` to a layout template, page component, or `gatsby-browser.js`.
+2. Then within your app entry file or site component, import it in. In Gatsby, you could choose to import it into a layout template (`layout.js`), page component (`index.js`), or `gatsby-browser.js`.
 
 ```jsx:title=src/components/layout.js
-import "typeface-your-chosen-font"
+import "fontsource-open-sans" // Defaults to weight 400 with all styles included.
+import "fontsource-open-sans/500.css" // Weight 500 with all styles included.
```

Comment:
Can you add in some context for these second two import examples?

For the last line in particular, I'm not sure what the comment means.

Reply:
This one is was a little bit of a challenge to keep concise but informative enough.
Each import example shows possible ways to import different weights and styles.

- First one is the default import which is weight 400 that includes normal and italic styles.
- The second one aims to show that you can specify weights, such as 500, by changing the import path. This still includes styles normal and italic.
- The third one shows the ability to specify the style normal or italics by changing the import path yet again. i.e. "fontsource-open-sans/600-italic.css" as an alternative to the example.

Reply:
Got it. So they don't have to do all three of these imports, they can choose whichever one they want to get the exact styling they need.

It might be helpful to split out the second two to be their own code block.

So you'd have the first one, showing a generic import. And then you'd have a text note saying something like "If you only want to import a particular font weight, you can also import a single file from the font package." and then have the second two examples.

That way the inline comments don't have to change but there's at least some context beforehand.

Reply:
I made the appropriate changes.